components/post_view/post/post.jsx

```diff
@@ -229,7 +229,7 @@ class Post extends React.PureComponent {
         }
 
         let rootUser = '';
-        if (this.hasSameRoot(this.props) && !fromBot) {
```

Comment:
This was added to fix https://mattermost.atlassian.net/browse/MM-15228 from what I can tell. Can you confirm that this isn't going to cause a regression in that ticket?

Reply:
Yup, it definitely causes an issue there. I'll fix that and this ticket together.